internal/ingress/defaults/main.go

```diff
@@ -107,6 +107,12 @@ type Backend struct {
 	// By default this list is empty
 	SkipAccessLogURLs []string `json:"skip-access-log-urls"`
 
+	// SkipAccessLogHTTPStatuses sets a list of HTTP statuses that should not appear in the NGINX access log
+	// The status strings provided are interpreted by an NGINX map as regex
+	// This is useful with statuses like 2xx and 3xx that make "complex" reading the logs
+	// By default this list is empty
+	SkipAccessLogHTTPStatuses []string `json:"skip-access-log-http-statuses,-"`
```

Comment:
why omit the json tag here? Can you unset the ",-"? We are going to need this tags in a future to marshall this config and split control plane and dataplane

Reply:
I don't recall why I would have done this. I have pushed a fix for this.